prisma/seed.ts

```diff
@@ -14,33 +14,46 @@ async function main() {
     password: hashedPassword,
   };
 
-  const isAdminExists = await prisma.admin.count({
+  const existedAdmin = await prisma.admin.count({
     where: { username: adminCreateInput.username },
   });
 
-  if (!isAdminExists) {
-    await prisma.admin.create({ data: adminCreateInput, select: null });
+  if (existedAdmin < 1) {
+    await prisma.admin.create({ data: adminCreateInput });
     console.log('the admin password is:', originalPassword);
   }
 
   const testFamily = { slug: 'test-family', name: faker.name.fullName() };
-  const isTestFamilyExists = await prisma.family.findFirst({
+  const existedFamily = await prisma.family.findFirst({
     where: { slug: testFamily.slug },
   });
 
   // eslint-disable-next-line fp/no-let
   let family: Family;
-  if (!isTestFamilyExists) {
+  if (existedFamily === null) {
     family = await prisma.family.create({ data: testFamily });
     console.log(
       `test family created, test family name is "${testFamily.name}"`,
     );
   } else {
-    family = isTestFamilyExists;
+    family = existedFamily;
   }
 
-  const isTestProjectExists = await prisma.project.findFirst();
-  if (!isTestProjectExists) {
+  const existedHouseholder = await prisma.householder.findFirst();
+
+  if (existedHouseholder === null) {
```

Comment:
you can get rid of ===, because if there is no householder then it'll be a falsy value.

```javascript
if (existedHouseholder){
....
}
```

Reply:
I guess it is more cleaner and understandable
because we know what types it could be

Reply:
You're using  typescript,
It specify output type therefore it's better to use short form of this